chevah/brink
# [#2640] Fix paver clean.

Problem
=======

`paver clean` fails on RHEL-4.
Also when a /tmp/pip_build_buildslave folder is found pip will abort

Paver clean is not removing the cached files.
This should be ok since all cached files are versioned and the same version will never change... with the exception of python binary dist which is not versioned.

Changes
=======

Always create a .pyc file . Paver clean is not called to often so this should not be a problem.
paver clean also looks for pip_build_* folders
add a paver.conf option to also clean python.

How to test
========

reviewers: @dumol 

check that changes make sense.
once this is merged we will need to manually copy the new version in server / compat and other repos
thanks!
